NitroxModel/Logger/Log.cs

```diff
@@ -142,6 +144,35 @@ private static void Setup()
             hierarchy.Root.AddAppender(fileAppender);
 
             hierarchy.Configured = true;
+
+        }
+
+        public static void RegisterUnityHandler()
+        {
+            Application.logMessageReceived += (condition, stacktrace, type) =>
+            {
+                switch (type)
+                {
+                    case LogType.Error:
+                        unityLog.Error(condition);
+                        break;
+                    case LogType.Assert:
+                        unityLog.Error($"Assertion failed: {condition}");
+                        break;
+                    case LogType.Warning:
+                        unityLog.Warn(condition);
+                        break;
+                    case LogType.Log:
+                        unityLog.Debug(condition);
+                        break;
```

Comment:
Agreed on LogType.Log. Too much information that's better handled by our logging methods on the spots we need. Best remove it.